comid/measurement_test.go

```diff
@@ -155,7 +164,84 @@ func TestUIntMkey_ToCBOR_ok(t *testing.T) {
 
 		assert.Nil(t, err)
 		assert.Equal(t, tv.expected, actual)
+	}
+}
+func TestUIntMkey_ToCBOR_notok(t *testing.T) {
+	tvs := []struct {
+		mkey     uint64
+		expected string
+	}{
+		{
+			mkey:     0,
+			expected: "unknown measurement key type: <nil>",
```

Comment:
this looks wrong.

Reply:
No I verified, if we do not set the right key, it would still Marshal Blank CBOR and would not fail the test!

Hence I put the check for validity of mkey inside Marshal CBOR for mkey

Reply:
What I meant is the logics here is wrong in the sense that we are not setting `tvs[i].mkey` anywhere.